bincompat-backward.whitelist.conf

```diff
@@ -223,6 +223,29 @@ filter {
     {
         matchName="scala.concurrent.impl.Promise.toString"
         problemName=MissingMethodProblem
+    },
+    // these next four can be removed once there is a fix for
+    // https://github.com/typesafehub/migration-manager/issues/147
+    {
+        matchName="scala.collection.Iterator#Leading#1.advance"
+        problemName=DirectMissingMethodProblem
+    },
+    {
+        matchName="scala.collection.Iterator#Leading#1.finish"
+        problemName=IncompatibleResultTypeProblem
+    },
+    {
+        matchName="scala.collection.Iterator#Leading#1.lookahead"
+        problemName=DirectMissingMethodProblem
+    },
+    {
+        matchName="scala.collection.Iterator#Leading#1.this"
+        problemName=DirectMissingMethodProblem
+    },
+    // ??? not sure what this is about
```

Comment:
Another instance of https://github.com/typesafehub/migration-manager/issues/147, fixed in MiMa master.